README.md

```diff
@@ -27,16 +29,78 @@ directory layout:
            |-- <command2>.py
 
 ```
+
+pycog3 also supports more advanced, multi-level structure when using the `-` field separator in the command name. 
+For example, defining:
+ * `commands\commanda.py` - maps to `!commanda`
+ * `commands\level1\commandb.py` - maps to `!level1-commandb`
+ * `commands\level1\level2a\commandc.py` - maps to `!level1-level2a-commandb`
+ * `commands\level1\level2b\leveln\commandz.py` - maps to `!level1-level2b-lebeln-commandz`
+ 
+
+```
+<bundle_name>
```

Comment:
It may be clearer to replace `<bundle_name>` here with `<bundle_directory>` since this directory doesn't need to be the same as the bundle name.